myriad-scheduler/src/main/java/org/apache/myriad/scheduler/NMPorts.java

```diff
@@ -23,12 +23,12 @@
 import java.util.Map;
 
 /**
- * Helper class for dynamically assigning ports to nodemanager
+ * Helper class for dynamically assigning ports to NodeManager
  */
 public class NMPorts implements Ports {
-  private static final String NM_RPC_PORT_KEY = "nm.rpc.port";
-  private static final String NM_LOCALIZER_PORT_KEY = "nm.localizer.port";
-  private static final String NM_WEBAPP_HTTP_PORT_KEY = "nm.webapp.http.port";
+  private static final String NM_RPC_PORT_KEY          = "nm.rpc.port";
+  private static final String NM_LOCALIZER_PORT_KEY    = "nm.localizer.port";
+  private static final String NM_WEBAPP_HTTP_PORT_KEY  = "nm.webapp.http.port";
   private static final String NM_HTTP_SHUFFLE_PORT_KEY = "nm.http.shuffle.port";
```

Comment:
Agreed, I will make that change.

On Wed, Jun 1, 2016 at 1:10 AM, Darin <notifications@github.com> wrote:

> In myriad-scheduler/src/main/java/org/apache/myriad/scheduler/NMPorts.java
> <https://github.com/apache/incubator-myriad/pull/71#discussion_r65302687>:
>
> >   */
> >  public class NMPorts implements Ports {
> > -  private static final String NM_RPC_PORT_KEY = "nm.rpc.port";
> > -  private static final String NM_LOCALIZER_PORT_KEY = "nm.localizer.port";
> > -  private static final String NM_WEBAPP_HTTP_PORT_KEY = "nm.webapp.http.port";
> > +  private static final String NM_RPC_PORT_KEY          = "nm.rpc.port";
> > +  private static final String NM_LOCALIZER_PORT_KEY    = "nm.localizer.port";
> > +  private static final String NM_WEBAPP_HTTP_PORT_KEY  = "nm.webapp.http.port";
> >    private static final String NM_HTTP_SHUFFLE_PORT_KEY = "nm.http.shuffle.port";
>
> Let's stick with one space before after "=", I know it's popular among
> some but the rest of the code is formatted that way.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/apache/incubator-myriad/pull/71/files/37b4df2dffec60b2825ed21754a56673eb195b03#r65302687>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AKSRgZSD88IBt483pny6bX4yY9BvlPCYks5qHRQ1gaJpZM4Ib_Am>
> .
>